codespell_lib/data/dictionary.txt

```diff
@@ -4731,7 +4731,9 @@ consrtuctor->constructor
 consrtuctors->constructors
 consrtucts->constructs
 consructor->constructor
+constain->constrain, contain,
 constained->constrained
+constaining->constraining
```

Comment:
Can we have containing, in fact please can you add contain* as a suggestion to all these typos.

Reply:
3033264